### ISSUE TITLE
Released version /3 of image modules for TF1.x

### ISSUE THREAD

**Issue by arnoegw** [tensorflow/hub#294]
[This announcement was simultaneously sent to the hub@tensorflow.org list and is stored as this GitHub issue for ease of reference.]

Version /3 of many TF Hub modules for [image feature extraction](https://tfhub.dev/s?module-type=image-feature-vector) and [classification](https://tfhub.dev/s?module-type=image-classification) with TensorFlow 1.x has just been published. See the list below for the changes in detail.

This is a maintenance update for users of TensorFlow 1.x before the TF Hub authors turn their attention to TensorFlow 2. (For a preview, see [this colab notebook](https://colab.research.google.com/github/tensorflow/hub/blob/master/examples/colab/tf2_image_retraining.ipynb) with GPU-accelerated image model fine-tuning using TF2, Keras, and TF-Hub. This will replace retrain.py.)

Today's changes for the TF1.x image modules are, in detail:

  * **Variable input size** (requested in [issue #11](https://github.com/tensorflow/hub/issues/11)). For for Inception, ResNet, and MobileNet V1 (except quantops), the height and width of the image input tensor are no longer fixed, so users can pass batches of images that are larger or smaller (within limits) than the default size. The output shape remains the same. The library symbol `hub.get_expected_image_size()` continues to provide the default size that the model was trained for.

    Under the hood, this uses attached messages to record the default size outside the input signature and hence requires library version `tensorflow-hub>=0.2.0` (released in Dec 2018).

  * **Fixing certain TFLite crashes with quantops** (see [issue #109](https://github.com/tensorflow/hub/issues/109)). The `mobilenet_v1*/quantops` modules now contain a workaround for a crash of quantized models in TFLite versions since last year related to an edge case about quantization bounds in `tf.mul()`.

  * **Faster moving averages in batch normalization.** This only affects advanced users who use `hub.Module("...", trainable=True, tags={"train"} if training ...)` to fine-tune image models; this does not affect retrain.py.

    Batch norm maintains moving averages of mean and variance across the dataset, for use at inference time. Previously, their decay coefficient (a.k.a. momentum) was set to the default values from TF-Slim, which were optimized for training for tens of thousands of steps. Transfer learning might get away with hundreds of steps, but only if batch norm's aggregates catch up fast enough.

    By default, the batch norm momentum is now set uniformly to 0.99 for Inception, ResNet, MobileNet V1/V2 (except quantops), and NASNet/PNASNet. This value is equal to the default in tf.keras and works reasonably well across various scenarios. The module documentation describes how power users can take explicit control to adjust it further using special signatures (consistently across modules).

    Side remark: For the `tf2-preview` modules released in March, this was already done. TF2 allows for a nicer way to adjust `batch_norm_momentum` and eventually other hparams: just pass some of them as an optional arguments when calling the module.

If you discover any issues with the new modules, please file [issues on GitHub](https://github.com/tensorflow/hub/issues/) as usual. For community support and advice, please use StackOverflow tags [tensorflow-hub](https://stackoverflow.com/questions/tagged/tensorflow-hub) or [tensorflow](https://stackoverflow.com/questions/tagged/tensorflow).
